client/state/themes/selectors/get-jetpack-upgrade-url-if-premium-theme.js

```diff
@@ -22,7 +23,9 @@ export function getJetpackUpgradeUrlIfPremiumTheme( state, themeId, siteId ) {
 		isThemePremium( state, themeId ) &&
 		! hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES )
 	) {
-		return `/checkout/${ getSiteSlug( state, siteId ) }/professional`;
+		return `/checkout/${ getSiteSlug( state, siteId ) }/${
+			shouldShowOfferResetFlow() ? PLAN_JETPACK_SECURITY_DAILY : 'professional'
```

Comment:
I think we should replace `PLAN_JETPACK_SECURITY_DAILY` with `PLAN_JETPACK_SECURITY_REALTIME` since Professional === Jetpack Security **Real-time** and Premium === Jetpack Security **Daily** in terms of features.

Reply:
backed up by the feature list here: 
https://jetpackp2.wordpress.com/2020/08/09/offer-reset-full-feature-list/

<img width="813" alt="Screen Shot 2020-09-15 at 12 27 16 PM" src="https://user-images.githubusercontent.com/2810519/93255496-dfa71400-f74e-11ea-892c-c8102479e3d3.png">

Reply:
And I did look at that table... my brain is muddy as well. Good catch, thanks!